node/packages/aws-lambda-otel-extension/test/unit/external/index.test.js

```diff
@@ -551,258 +252,26 @@ describe('external', () => {
         reports = await waitForReports();
         expect(reports.map(([name]) => name)).to.deep.equal(['logs']);
-        expect(reports[0][1].map(({ Body: body }) => body)).to.deep.equal(
+        expect(reports[0][1].messages.map(({ message }) => message)).to.deep.equal(
           logs.map(({ record }) => record)
         );
 
-        await sendTelemetryData();
-
         emitPlatformRuntimeDone();
 
         await new Promise((resolve) => testUtils.listenerEmitter.once('next', resolve));
 
         reports = resolveReports();
-        expect(reports.map(([name]) => name)).to.deep.equal(['metrics', 'traces']);
+        expect(reports.map(([name]) => name)).to.deep.equal([]);
 
         // Shutdown
         emitShutdown();
 
         emitPlaftormEndAndReport();
-
-        reports = await waitForReports();
-        expect(reports.map(([name]) => name)).to.deep.equal(['metrics']);
       }
     );
-    extensionProcess.stdout.on('data', (line) => stdoutLines.push(line));
-    await promise;
-  });
-
-  it('should handle race condition where telemetry data arrives before event data', async () => {
-    const { extensionProcess, promise } = extensionProcessHandler(
-      { region, functionName },
-      async (testUtils) => {
-        emitter = testUtils.emitter;
-
-        emitExtensionReady();
-
-        // Invocation
-        requestId = uuidv4();
-
-        emitInvoke();
-        emitPlatformStart();
-
-        const responseData = {
-          statusCode: 200,
-          body: '{"result":"ok","filename":"/var/task/callback.js"}',
-        };
-        await sendTelemetryData({
-          responseEventPayload: {
-            responseData,
-            errorData: null,
-            executionId: requestId,
-          },
-        });
-        await wait(20);
-
-        const requestData = { foo: 'bar' };
-        await sendEventData({
-          requestEventPayload: {
-            requestData,
-            timestamp: 1657823430963,
-            executionId: requestId,
-          },
-        });
-
-        emitPlatformRuntimeDone();
-
-        await new Promise((resolve) => testUtils.listenerEmitter.once('next', resolve));
-
-        reports = resolveReports();
-        expect(reports.map(([name]) => name)).to.deep.equal([
-          'metrics',
-          'traces',
-          'request',
-          'response',
-        ]);
-        expect(reports[2][1].requestData).to.deep.equal(requestData);
-        const responseReport = reports[3][1];
-        expect(responseReport.responseData).to.deep.equal(responseData);
-
-        // Shutdown
-        emitShutdown();
-
-        emitPlaftormEndAndReport();
-
-        reports = await waitForReports();
-        expect(reports.map(([name]) => name)).to.deep.equal(['metrics']);
-      }
-    );
-    extensionProcess.stdout.on('data', (line) => stdoutLines.push(line));
-    await promise;
-  });
-
-  it('should handle case where platform.start arrives before invoke', async () => {
```

Comment:
This test is quite important

Reply:
Added these tests back since I moved the go tests will no longer share this file 👍